module_27_Python/3_data.py

```diff
@@ -0,0 +1,58 @@
+"""
+Задача 3. Дата
+Реализуйте класс Date, который должен:
+- проверять числа даты на корректность;
+- конвертировать строку даты в объект класса Date, состоящий из соответствующих
+    числовых значений дня, месяца и года.
+Оба метода должны получать на вход строку вида ‘dd-mm-yyyy’.
+При тестировании программы объект класса Date должен инициализироваться
+исключительно через метод конвертации, например:
```

Comment:
Python doesn't have built-in private methods or deleted function, so you can hide the constructor or make it unusable outside of the class
Here is a possible solution
https://stackoverflow.com/questions/8212053/private-constructor-in-python